cmd/ncproxy/store.go

```diff
@@ -0,0 +1,84 @@
+package main
+
+import (
+	"context"
+
+	"github.com/containerd/containerd/errdefs"
+	"github.com/pkg/errors"
+	bolt "go.etcd.io/bbolt"
+)
+
+type computeAgentStore struct {
+	DB *bolt.DB
+}
+
+func newComputeAgentStore(db *bolt.DB) *computeAgentStore {
+	return &computeAgentStore{DB: db}
+}
+
+func (n *computeAgentStore) get(ctx context.Context, key string) (result string, err error) {
```

Comment:
Maybe I'm missing something, but is this function only called from test code?

Reply:
Yes that's correct. Would you prefer I remove it?

Reply:
It should at least be defined in test code in this case.

Reply:
what do you mean?

Reply:
All the files without `_test.go` suffix are product code, and shouldn't have code in them that is only used for tests. If this function is only used for test code then it should be in a `_test.go` file.